APKBUILD

```diff
@@ -25,8 +25,6 @@ package() {
   rm -rf "$pkgdir"/usr/glibc-compat/lib/gconv
   rm -rf "$pkgdir"/usr/glibc-compat/lib/getconf
   rm -rf "$pkgdir"/usr/glibc-compat/lib/audit
-  rm -rf "$pkgdir"/usr/glibc-compat/lib/*.a
-  rm -rf "$pkgdir"/usr/glibc-compat/include
```

Comment:
When there's a `-dev` package being built, `abuild` functions look for `*.a` and `*.h` files and only include them in that package. They're excluded from the other `glibc` and `glibc-bin` packages.